project/AppDependencies.scala

```diff
@@ -8,9 +8,7 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc" %% "bootstrap-backend-play-28" % bootstrapPlayVersion,
```

Comment:
Please bring other dependencies up-to-date with reference to [the catalogue](https://catalogue.tax.service.gov.uk/repositories/api-platform-outbound-soap)
You'll need to make changes also, in `build.sbt`, `plugins.sbt` and `build.properties` to get them all